src/DataWrangling/ECCO/ECCO_metadata.jl

```diff
@@ -12,6 +12,7 @@ import Oceananigans.Fields: location
 import ClimaOcean.DataWrangling: all_dates, metadata_filename, download_dataset,
                                  default_download_directory, dataset_temperature_units,
                                  short_name, dataset_latitude_extent
+import ClimaOcean.DataWrangling.ECCO: download_ECCO_cache
```

Comment:
1/ is this needed?
2/ do we need to import it or using it?

I believe it's not needed since it was defined in ECCO.jl, right?